Symfony/CS/ConfigInterface.php

```diff
@@ -49,9 +51,18 @@ public function getFinder();
     public function getLevel();
 
     /**
-     * Returns the fixers to run.
+     * Set the fixers to be used.
+     *
+     * @param string[] $fixers
+     *
+     * @return ConfigInterface
+     */
+    public function fixers(array $fixers);
+
+    /**
+     * Returns the fixers.
      *
-     * @return array A list of fixer names
+     * @return FixerInterface[] A list of fixers
```

Comment:
Same problem as mentioned here: https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1248#discussion_r32312964

setter takes both `string[]` and `FixerInterface[]`, and this is the same for getter output.